Make uv run environment propagation the default (#53060)
ray-project/ray: doc/source/ray-core/handling-dependencies.rst

  run in via `py_executable`. It could even be a shell script that is stored in `working_dir` if you are trying to wrap
  multiple processes in more complex ways.

.. note::

>   The uv environment will be inherited by all children tasks and actors. If you want to mix e.g. `pip`

Reviewer: ```suggestion
  The uv environment is inherited by all children tasks and actors. If you want to mix environments, for example,  `pip`
```

>   runtime environments with `uv run` you will need to set the Python executable back to an executable

Reviewer: ```suggestion
  runtime environments with `uv run`, you need to set the Python executable back to an executable
```

>   that is not running in the isolated uv environment like so:

Reviewer: ```suggestion
  that's not running in the isolated uv environment like the following:
```

  .. code-block:: toml

    [project]

    name = "test"

    version = "0.1"

    dependencies = [
      "emoji",
      "ray",
      "pip",
      "virtualenv",
    ]

  .. testcode::
    :skipif: True

    import ray

    @ray.remote(runtime_env={"pip": ["wikipedia"], "py_executable": "python"})
    def f():
        import wikipedia
        return wikipedia.summary("Wikipedia")

    @ray.remote
    def g():
        import emoji
        return emoji.emojize('Python is :thumbs_up:')

    print(ray.get(f.remote()))
    print(ray.get(g.remote()))

>   While the above pattern can be useful for supporting legacy applications, it is recommended to

Reviewer: ```suggestion
  While the above pattern can be useful for supporting legacy applications, the Ray Team recommends
```

>   also use uv for tracking nested environments. You can do this by creating a separate

Reviewer: ```suggestion
  also using uv for tracking nested environments. You can use this approach by creating a separate
```
